GenderPayGap.WebUI/Controllers/SubmitController.CheckData.cs

```diff
@@ -185,6 +191,14 @@ public async Task<IActionResult> CheckData(ReturnViewModel postedReturnViewModel
                 ModelState.Remove(nameof(postedReturnViewModel.LateReason));
                 ModelState.Remove(nameof(postedReturnViewModel.EHRCResponse));
             }
+            
+            // Don't mark submission as late if the reporting year has been excluded from late flag enforcement (eg. 2019/20 due to COVID-19)
+            if (Global.ReportingStartYearsToExcludeFromLateFlagEnforcement.Contains(stashedReturnViewModel.AccountingDate.Year))
+            {
+                ModelState.Remove(nameof(postedReturnViewModel.LateReason));
+                ModelState.Remove(nameof(postedReturnViewModel.EHRCResponse));
+                postedReturn.IsLateSubmission = false;

```

Comment:
This needs to be `postedReturn` rather than `postedReturnViewModel` to correctly not mark submission as late